main.go

```diff
@@ -135,78 +224,76 @@ func wrapMain() error {
 
 	e := os.Environ()
 	e = append(e, fmt.Sprintf("GOOGLE_APPLICATION_CREDENTIALS=%s", keyPath))
-	runner := NewEnviron(workspace.Path, e, os.Stdout, os.Stderr)
+	wd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("Error while getting working directory: %s\n", err)
+	}
+	runner := NewEnviron(wd, e, os.Stdout, os.Stderr)
 
-	err = runner.Run(vargs.GCloudCmd, "auth", "activate-service-account", "--key-file", keyPath)
+	err = runner.Run(gcloudCmd, "auth", "activate-service-account", "--key-file", keyPath)
 	if err != nil {
 		return fmt.Errorf("Error: %s\n", err)
 	}
 
-	err = runner.Run(vargs.GCloudCmd, "container", "clusters", "get-credentials", vargs.Cluster, "--project", vargs.Project, "--zone", vargs.Zone)
+	err = runner.Run(gcloudCmd, "container", "clusters", "get-credentials", c.String("cluster"), "--project", project, "--zone", c.String("zone"))
 	if err != nil {
 		return fmt.Errorf("Error: %s\n", err)
 	}
 
 	data := map[string]interface{}{
-		// http://readme.drone.io/usage/variables/#string-interpolation:2b8b8ac4006be88c769f5e3fd99b009a
-		"BUILD_NUMBER": build.Number,
-		"COMMIT":       build.Commit,
-		"BRANCH":       build.Branch,
-		"TAG":          "", // How?
+		"BUILD_NUMBER": c.String("drone-build-number"),
+		"COMMIT":       c.String("drone-commit"),
+		"BRANCH":       c.String("drone-branch"),
+		"TAG":          c.String("drone-tag"),
 
 		// https://godoc.org/github.com/drone/drone-plugin-go/plugin#Workspace
-		"workspace": workspace,
-		"repo":      repo,
-		"build":     build,
-		"system":    system,
+		// TODO do we really need these?
```

Comment:
I don't know if these are needed -- it would allow people to reference variables in kube templates that aren't explicitly called out in the pipeline configuration. I think we should not allow that. If a pipeline author wants to reference this data, they should explicitly pass it in as a var in `.drone.yml`